include/meta/meta.hpp

```diff
@@ -1797,9 +1797,10 @@ namespace meta
         /// \endcond
 
     /// Return a new \c meta::list constructed by transforming all the
-    /// elements in \p L with the unary Invocable \p Fn. \c transform can
-    /// also be called with two lists of the same length and a binary
-    /// Invocable, in which case it returns a new list constructed with the
+    /// elements in \p L as the first argument with the unary Invocable \p Fn as
+    /// the second argument. 
```

Comment:
This is ambiguous. We're talking about invoking `p`, and also talking about argument position, but we're not talking about the positions of the arguments to `p`; we're talking of the position of arguments to `transform`. Maybe:

> "transforming all the elements in L (the first argument)  with the unary Invocable Fn (the second argument)."